assets/dev/js/editor/regions/panel/pages/menu/menu.js

```diff
@@ -67,6 +67,7 @@ PanelMenu.addAdminMenu = () => {
 		],
 	}, { at: 0 } );
 
+	// TODO: Move to the Notes module
```

Comment:
why didn't you move it already?

Reply:
Very out of scope of the current task.

Reply:
but you're already here. If it's easy to implement, I think you should add it here

Reply:
Not super complicated, but also will require some development and some testing to do.
I don't want to mix with the current task. Created a new one for it.